backend/src/main/kotlin/org/icpclive/controllers/TeamViewController.kt

```diff
@@ -1,26 +1,51 @@
 package org.icpclive.controllers
 
 import kotlinx.coroutines.delay
-import org.icpclive.api.TeamViewPosition
-import org.icpclive.api.TeamViewSettings
-import org.icpclive.api.TeamViewWidget
+import kotlinx.coroutines.flow.first
+import org.icpclive.api.*
 import org.icpclive.data.DataBus
 import org.icpclive.data.Manager
 import kotlin.time.Duration.Companion.seconds
 
 class TeamViewController(val manager: Manager<TeamViewWidget>, val position: TeamViewPosition) :
-    SingleWidgetController<TeamViewSettings, TeamViewWidget>(TeamViewSettings(), manager, ::TeamViewWidget) {
-    override suspend fun createWidgetAndShow(settings: TeamViewSettings) {
+    SingleWidgetController<ExternalTeamViewSettings, TeamViewWidget>(
+        ExternalTeamViewSettings(),
+        manager,
+        { TeamViewWidget(OverlayTeamViewSettings()) }) {
```

Comment:
I think this lambda shouldn't be used, as we override construct. So, maybe we should throw an exception here? 

To be honest, I don't like having both this lambda and override, but I have no good idea what to do with it right now.